addons/danger/functions/fnc_tacticsSuppress.sqf

```diff
@@ -22,6 +22,13 @@ params ["_unit", "_target", ["_units", []], ["_delay", 17]];
 // find target
 _target = _target call CBA_fnc_getPos;
 
+// exit with flank if visible distance is far actual target (within one grid : 71 * 71 meters)
+private _targetASL = ATLtoASL _target;
+private _vis = lineIntersectsSurfaces [(eyePos _unit) vectorAdd [0, 0, 12], _targetASL vectorAdd [0, 0, 2.5], _unit, objNull, true, 1, "FIRE", "GEOM"];
+if (_vis isNotEqualTo [] && {((_vis select 0) select 0) vectorDistanceSqr _targetASL > 5041}) exitWith {
```

Comment:
didn't we want to filter this a bit more?
so using `_vis findIf {(_x select 0) isKindOf "Building"};` is what we wanted to do I think

Reply:
Will investigate now.

Reply:
Added